keyboards/haverworks/theseus75/config.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Moritz Plattner (@ebastler), Alex Havermale (@haversnail)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Defines for configuring the serial driver for split comms (see https://docs.qmk.fm/drivers/serial) */
+#define SERIAL_USART_DRIVER SD3    // USART 3
+
+/* Defines for required pins */
+#define ID_PIN A13
+#define USBSW_PIN A14     // Switches the hub input/output with the MUXes
+#define PSW_PIN A15       // Turns the power routing to the USB-C port on/off
+#define BUS_B_PIN C13     // Tells the hub to be bus-powered or self-powered (which in turn tells the clients to use 100 mA or 500 mA)
+#define USB_VBUS_PIN C15  // Only the master side will have VBUS present at power-up (used to determine master/slave)
+#define USBPD_1_PIN F0
+#define USBPD_2_PIN F1
+
+/* Additional defines for managing power and state scross split */
+#define DISABLE_BUS_POWER_MODE TRUE
+#define KB_STATE_SYNC_INTERVAL 500
+#define USBPD_ALLOWANCE_CHECK_INTERVAL 100
+#define SPLIT_TRANSACTION_IDS_KB RPC_ID_KB_STATE
+
+/* Defines for the RGB matrix */
+#define INDICATOR_MAX_BRIGHTNESS 255
```

Comment:
Was there a reason this exceeds the RGB Matrix maximum brightness set in `keyboard.json`?
```suggestion
#define INDICATOR_MAX_BRIGHTNESS 128
```

Reply:
IIRC, the brightness on the rest of the LEDs were lowered to reduce total power consumption by the board, to maximize power to the downstream USB hub and any connected peripheral (@ebastler keep me honest); however, the LED for the indicator we just kept at max brightness. (There'll be a custom VIA menu to modify the indicator's H/S/V directly anyways.)

Reply:
So, no reason other than to make sure it's plenty bright. ;)

Reply:
@waffle87 any concerns with the above approach? If not, happy to resolve the thread 👍